test/python/fe_api/sdpa/test_sdpa_bwd_sm80.py

```
@@ -546,3 +556,112 @@ def run(lse):
+            is_causal=True,
+            window_size=window,
+            causal_bottom_right=bottom_right,
+            scale_softmax=scale,
+            deterministic=det,
+        )
+
+    a, bb = run(True), run(True)
+    for key in ("dq_tensor", "dk_tensor", "dv_tensor"):
+        assert torch.equal(a[key], bb[key]), key
+    nd = run(False)
+    _, dq_ref, dk_ref, dv_ref = _ref_grads(q, k, v, do, is_causal=True, window_left=w, scale=scale, causal_bottom_right=bottom_right)
```

Ruff (0.16.3)

[warning] Unpacked variable `dk_ref` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)